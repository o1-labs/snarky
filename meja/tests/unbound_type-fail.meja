type t = unknown_type
