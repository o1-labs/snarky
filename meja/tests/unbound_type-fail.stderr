File "tests/unbound_type-fail.meja", line 1, characters 9-21:
Error: Unbound type constructor unknown_type.
